legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabReviewApi.java

```diff
@@ -122,8 +157,22 @@ else if (revisionIds instanceof Set)
             {
                 // if no revision ID is specified we will use the default merge request API from Gitlab to take advantage of the filter
                 MergeRequestFilter mergeRequestFilter = new MergeRequestFilter()
-                        .withProjectId(gitLabProjectId.getGitLabId())
                         .withState(mergeRequestState);
+
+                if (projectId != null)
+                {
+                    mergeRequestFilter.setProjectId(projectId);
+                }
+
+                if (assignedToMe != null && assignedToMe)
```

Comment:
This logic should be removed once assignedToMe is boolean (rather than Boolean).